recipes/autogen/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "5.18.7" %}
+
+package:
+  name: autogen
+  version: {{ version }}
+
+source:
+  fn: autogen-5.18.7.tar.xz
+  url: http://ftpmirror.gnu.org/autogen/autogen-5.18.7.tar.gz
+  sha1: 83dc95438470447b13f9b958871d7c3b7a868733
+
```

Comment:
Add a `build` section with a build `number`. (See the example recipe directory for more info.)

You can also add a `skip: True  # [win]` since you mentioned that there are no Windows builds.

PS: When committing try to add `[skip appveyor]` to your message to prevent it from starting.